app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { sendTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+
+app.use(express.json());
+
```

Comment:
I would look to add some error handling here to deal with a 500 error (server error) - you don't need to test for this but it should be included